packages/@webex/internal-plugin-scheduler/src/scheduler/scheduler.js

```diff
@@ -1,39 +1,31 @@
+import btoa from 'btoa';
 import {WebexPlugin} from '@webex/webex-core';
 
```

Comment:
thanks, I will try to reuse [base64.js](https://github.com/webex/webex-js-sdk/blob/master/packages/%40webex/common/src/base64.js) utilities file, instead of btoa.